src/test/java/io/supertokens/test/DotStartedFileTest.java

```diff
@@ -135,15 +137,36 @@ public void dotStartedFileNameAndContentTest() throws Exception {
 
         try (InputStream is = new FileInputStream(dotStartedNameAndContent[0].getPath());
                 BufferedReader buf = new BufferedReader(new InputStreamReader(is))) {
-
             String line = buf.readLine();
             assertEquals(line, Long.toString(ProcessHandle.current().pid()));
-
+            line = buf.readLine();
+            assertEquals(line, Config.getConfig(process.main).getBasePath());
         }
 
         process.kill();
         assertNotNull(process.checkOrWaitForEvent(ProcessState.PROCESS_STATE.STOPPED));
 
+        // Ensure that base_path is set in .started file
+        Utils.setValueInConfig("base_path", basePathCheck);
+
+        process = TestingProcessManager.start(args);
+        assertNotNull(process.checkOrWaitForEvent(ProcessState.PROCESS_STATE.STARTED));
+
+        dotStartedNameAndContent = loc.listFiles();
+        assert dotStartedNameAndContent != null;
+        assertEquals(1, dotStartedNameAndContent.length);
+        assertEquals(dotStartedNameAndContent[0].getName(), hostPortNameCheck);
+
+        try (InputStream is = new FileInputStream(dotStartedNameAndContent[0].getPath());
+                BufferedReader buf = new BufferedReader(new InputStreamReader(is))) {
+            String line = buf.readLine();
+            assertEquals(line, Long.toString(ProcessHandle.current().pid()));
+            line = buf.readLine();
+            assertEquals(line, Config.getConfig(process.main).getBasePath());
```

Comment:
```suggestion
            assertEquals(line, Config.getConfig(process.main).getBasePath());
            assertEquals(line, basePathCheck);
```

Reply:
I added the extra check just to make sure that the config change above is coming into affect.